### PR TITLE
begin home page render data from api/current_user

### PR DESCRIPTION
**Pull request by allen138**

### CODE REVIEW

File: client/src/components/RegisterForm.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import Wrapper from "./Wrapper";
 import Joi from "joi-browser";
 import Form from "./Form";
 import * as userService from "../services/userService";
+//import axios from "axios"
```
Comment (DChan0319) [inline client/src/components/RegisterForm.js:6]: Do we need this?

File: client/src/pages/HomePage.js [hunk 2]
```diff
@@ -5,19 +5,16 @@ import Footer from "../components/Footer";
...
-    }
+    state = {
+        user: {}
+    };
 
     componentDidMount() {
-        axios.get("http://localhost:5000/api/current_user").then((response) => {
+        axios.get("/api/current_user").then((response) => {
             const currentUser = response.data;
+            console.log(currentUser);
```
Comment (DChan0319) [inline client/src/pages/HomePage.js:15]: do we need this?

File: client/src/pages/HomePage.js [hunk 3]
```diff
@@ -5,19 +5,16 @@ import Footer from "../components/Footer";
...
             console.log("MOUNT CALL + " + Object.keys(currentUser))
-            if(currentUser){
+            if (currentUser) {
                 this.setState({
                     user: currentUser
                 })
```
Comment (DChan0319) [inline client/src/pages/HomePage.js:20]: semicolon

File: controller/passport.js [hunk 4]
```diff
@@ -42,8 +42,9 @@ passport.use(
             authMode: github,
             authModeID: profile.id
           }).then(function (user) {
+            console.log("reading this line...")
```
Comment (DChan0319) [inline controller/passport.js:45]: do we need this?

File: controller/passport.js [hunk 5]
```diff
@@ -42,8 +42,9 @@ passport.use(
...
             console.log(user.id);
```
Comment (DChan0319) [inline controller/passport.js:46]: same here do we need?

**Review by phelanmccall — APPROVED** [review]

File: client/src/pages/HomePage.js (hunk 3 above)
Comment (jmckenney) [reply client/src/pages/HomePage.js:20]: and colons are important.

File: client/src/pages/HomePage.js [hunk 6]
```diff
@@ -5,19 +5,16 @@ import Footer from "../components/Footer";
...
-            user: null
-        }
-    }
+    state = {
+        user: {}
+    };
 
     componentDidMount() {
-        axios.get("http://localhost:5000/api/current_user").then((response) => {
+        axios.get("/api/current_user").then((response) => {
```
Comment (jmckenney) [inline client/src/pages/HomePage.js:13]: now we're talking.

File: client/src/pages/HomePage.js [hunk 7]
```diff
@@ -27,18 +24,23 @@ class Home extends Component {
...
             <div>
-                <HomeNav/>
+                <HomeNav />
                 <Wrapper>
-                   {
-                       display
-                   }
+                    {display}
+                    <div className="userInfo">
+                        <img src={this.state.user.avatar} alt={this.state.user.firstName} width="150" height="150"/>
```
Comment (jmckenney) [inline client/src/pages/HomePage.js:39]: Looks like a job for object destructing. 
```javascript
const { avatar, firstName } = this.state;
```
Then you can use ust `avatar` and `firstName` in your jsx.

**Pull request merged**